examples/sync/src/bin/server.rs

```diff
@@ -485,9 +497,9 @@ fn parse_config() -> Result<Config, Box<dyn std::error::Error>> {
         .arg(
             Arg::new("db")
                 .long("db")
-                .value_name("any|immutable")
-                .help("Database type to use. Must be `any` or `immutable`.")
-                .default_value("any"),
+                .value_name("fixed|variable|immutable")
```

Comment:
should we use "any::fixed" and "any::variable"?  (I worry we'll have to eventually add fixed/variable variants of other dbs, say, current::).